.github/scripts/highlights.mjs

```diff
@@ -0,0 +1,76 @@
+// @ts-check
+import * as process from 'node:process';
+import { Octokit } from '@octokit/core';
+import { output } from './util.mjs';
+
+const {
+  GITHUB_TOKEN = '',
+  PR_LIST = '',
+  owner = 'mongodb',
+  repo = 'node-mongodb-native'
+} = process.env;
+if (GITHUB_TOKEN === '') throw new Error('GITHUB_TOKEN cannot be empty');
+
+const octokit = new Octokit({
+  auth: GITHUB_TOKEN,
+  log: {
+    debug: msg => console.error('Octokit.debug', msg),
+    info: msg => console.error('Octokit.info', msg),
+    warn: msg => console.error('Octokit.warn', msg),
+    error: msg => console.error('Octokit.error', msg)
+  }
+});
+
+const prs = PR_LIST.split(',').map(pr => {
+  const prNum = Number(pr);
+  if (Number.isNaN(prNum))
+    throw Error(`expected PR number list: ${PR_LIST}, offending entry: ${pr}`);
+  return prNum;
+});
+
+/** @param {number} pull_number */
+async function getPullRequestContent(pull_number) {
+  const startIndicator = 'RELEASE_HIGHLIGHT_START -->';
+  const endIndicator = '<!-- RELEASE_HIGHLIGHT_END';
+
+  let body;
+  try {
+    const res = await octokit.request('GET /repos/{owner}/{repo}/pulls/{pull_number}', {
```

Comment:
Should this be `'GET /repos/{owner}/{repo}/pull/{pull_number}' ? I'm seeing a 404 in the test action for the highlights and they don't show up in the sample release PR.

Reply:
https://docs.github.com/en/rest/pulls/pulls?apiVersion=2022-11-28#list-pull-requests

It's "pulls" plural, the sample runs on the fork are trying to get double digit PRs on the driver repo instead of on the fork. Our low number "pulls" are actually the disabled github issues. I have a run in there somewhere where it worked, hence why I have owner/repo as variables, just gotta find it. 

The good thing is that we always have the fallback of writing the notes ourselves, if we are unable to gather the notes from each PR, we can just paste it altogether ourselves into the release PR before merging and it will end up in the GH release.